.github/workflows/claude.yml

```
@@ -1,34 +1,29 @@
       (github.event_name == 'pull_request_review' && contains(github.event.review.body, '@claude')) ||
       (github.event_name == 'issues' && (contains(github.event.issue.body, '@claude') || contains(github.event.issue.title, '@claude')))
-    
     runs-on: ubuntu-latest
     permissions:
       contents: read
       pull-requests: read
       issues: read
       id-token: write
-      actions: read
-    
+      actions: read # Required for Claude to read CI results on PRs
```

YAMLlint (1.37.1)

[warning] too few spaces before comment: expected 2

(comments)